### DISCUSSION TITLE
Log file doesn't exist

### DISCUSSION THREAD

**Discussion by fabuloudy** [apache/airflow#30274] [Q&A]
Hello, guys! 
I have a problem. A lot of DAGs running on my server: distributed among clusters, processing a lot of data and so on. And it works stable. But one DAG once a week failed with message "Log file doesn't exist". I thought there might be problem connected with sending data between tasks. In the end I combined these tasks. But periodically this problem is showed again and again once a week. Do you have any suggestions where I can find the reason?

**Comment by kobethuwis** [comment]
How are you deploying Airflow? I've experienced this error due to a few different reasons: in short, it is exactly what it says 🤓 Airflow can't find you logfile: where are you saving them? 

For example, I'm saving them in Amazon S3, using a specific provider and the approach mentioned [here](https://airflow.apache.org/docs/apache-airflow-providers-amazon/stable/logging/s3-task-handler.html). When a task is still running however, the client falls back to a local log. Since Airflow is deployed in a private kubernetes cluster behind some ingresses, the webserver couldn't serve the worker's logs. In order to fix that I specified an extra argument in my chart:

```
extraEnv: |
  - name: AIRFLOW__CORE__HOSTNAME_CALLABLE
    value: 'airflow.utils.net.get_host_ip_address'
```
